Jackee Rohrich/index.html

```diff
@@ -0,0 +1,101 @@
+        </ul>
+        </div>
+    </body>
+
+</html>
+<html lang="en">
+<head>
+<link rel="stylesheet" href="./Specs.css">
+<script defer="defer" src="./components/tabs.js"></script>
+</header>
+    <div class="name">
+    <h1>Jackee Rohrich</h1>
+</div>
+
+<div id="Welcome!">
+<h1 class="container">Welcome!</h1>
+<p class="p">This is just a little page to tell you a little bit about me.</p>
+</div>
+
+<div id="About Me">
+
+<h1 class="container">About Me</h1>
+<img src="./images/1.jpg"" width="275" height="265" />
+<img src= "./images/2.jpg" width="275" height="265" />
+<img src="./images/3.jpg" "width="275" height="265" />
+<p> I am 34, I live in Bismarck, ND., with my Boyfriend, Jeff. I also am a Mother to Ella who is 10, and Cohen who is 8. Before joining Lambda School I was a Cosmetologist who owned my own business. 
+</p>    
+</div>
+<div id="Projects">
+<h1 class="container">Projects</h1>
+<!-- <p>None Currently</p>  -->
+</html>
+
+		<div class="Section Section--bordered">
+			<div class="Section__full">
+				<!-- <div class="Section__title">Tab Menu</div> -->
+				<div class="Tabs">
+					<div class="List">
+						<div class="List__item Tabs__link" data-tab="1">Captain Jack Sparrow</div>
+						<div class="List__item Tabs__link" data-tab="2">Jack Skellington</div>
+						<div class="List__item Tabs__link" data-tab="3">Pizza Obssessed Zombie</div>
+					</div>
+
+					<div class="Tabs__item" data-tab="1">
+						<div class="Box Box--placeholder Box--grey">
+                            <div class="Box__title">Captain Jack Sparrow</div>
+                            <div><img src="./images/5.jpg""width="275" height="265"/></div>
+							<div class="Box__description">Blah blah blah</div>
+						</div>
+					</div>
+					<div class="Tabs__item" data-tab="2">
+						<div class="Box Box--placeholder Box--blue">
+                            <div class="Box__title">Jack Skellington</div>
+                            <div><img src="./images/6.jpg""width="275" height="265"/></div>
+							<div class="Box__description">Blah blah blah blah blah blah</div>
+						</div>
+					</div>
+					<div class="Tabs__item" data-tab="3">
+						<div class="Box Box--placeholder Box--green">
+                            <div class="Box__title">Pizza Obssessed Zombie</div>
+                            <div><img src="./images/4.jpg""width="275" height="265"/></div>
+							<div class="Box__description">Blah</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+
+	</body>
+</html>
+</div>
+<div id="Contact">
+<h1 class="container">Contact</h1>
+<address>
+  <a href="mailto:Jacquelynn.A.Rohrich@gmail.com">Click Here to Email</a>.<br> 
+        </address>
+<br><br><br><br><br><br><br><br><br><br><br><br><br>
+<br><br><br><br><br><br><br><br><br><br><br><br><br>
+<!-- // I did breaks to test the function of my scroll menu buttons -->
+</div>
+
+</body>
+</html>
+</div>
+<footer class="footer"
+<h2><br> </h2>
```

Comment:
maybe add some margin to the footer instead of having this tag here with a break tag in it. Or maybe I'm misunderstanding the intent.